hpy/devel/include/hpy/hpydef.h

```diff
@@ -10,7 +10,7 @@ extern "C" {
 #include "hpy/autogen_hpyslot.h"
 #include "hpy/cpy_types.h"
 
-typedef void* (*HPyCFunction)();
+typedef void* (*HPyCFunction)(void);
```

Comment:
Am I using a different compiler to everyone else? My default compiler (gcc 9.4) complained about this ...

Reply:
Could be (I'm on Mac and compiling with LLVM).
However, that is semantically not the same: specifying `void` really means that the function doesn't have arguments while leaving the argument list empty means that it is not specified at this point. I hope that doesn't make problems elsewhere.

Reply:
The function declaration should match the function definition. Why can't we specify the argument list when we know what it is meant to be?